nixos/platform/network.nix

```diff
@@ -434,6 +441,21 @@ in
 
                 ${sysctlSnippet}
               '';
+              reload = ''
+                IFACE=${iface.layer2device}
+
+                # Set underlay address for virtual interface ${iface.layer2device}
+                ip link set $IFACE type vxlan local ${fclib.underlay.loopback}
```

Comment:
we should re-apply all settings identical to the add statement above. maybe also make a statement here to keep this in sync (or can we share the code?)

Reply:
I've tested this just now, and it's not possible to change the VNI or the destination port for a VXLAN interface after it's been created -- the only thing we can actually change is VTEP IP address (as I have here already).

Is it worth keeping the VXLAN interface reload functionality? Or should we drop this seeing as we can only change a subset of the relevant settings at runtime?

Reply:
Yeah. I'm fine with this. Maybe mention this in the comment?

Reply:
Fixed in [ef6cba1](https://github.com/flyingcircusio/fc-nixos/pull/951/commits/ef6cba11b056d292ddf41d9ff34371c36eef6c46).